app/controllers/api/v1/ingredients_search_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::IngredientsSearchController < ApplicationController
+  respond_to :json
+
+  def index
+    respond_with IngredientFinder.fetch_ingredients(params[:search])
```

Comment:
@jmejia is this application of the service and the helper an accurate representation of what you were trying to teach us in the small group session the other day?